testing/functional/tests/cypress/integration/01-student/04-application-page.spec.ts

```diff
@@ -176,18 +176,18 @@ describe("Application Page", () => {
   });
 
   it("Verify that user redirect to Personal Information page from Program page in application form.", () => {
-    cy.fixture("draftApplicationData").then((testData) => {
+    cy.fixture("draftApplicationData").then((_testData) => {
```

Comment:
Please rename `_testData` to `testData`.

Reply:
Updated fixture name.